vsce compare changes in dev (#8250)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/cloud-cli-installation.md

:::note CLI compatibility

> The dbt CLI is a <Constant name="dbt_platform" /> tool available to users on any [available plan](https://www.getdbt.com/pricing). If your dbt Cloud project uses Fusion as its engine, the platform CLI will automatically use Fusion for remote execution — no separate Fusion install is required.

Reviewer: Hey @mirnawong1 had a separate PR for this section that we were workshopping [here](https://github.com/dbt-labs/docs.getdbt.com/pull/8866)

I think it might be beneficial, rather than opening the page with a long callout, let's move all of this info down to the prerequisites and explain it there in detail. That would provide a more consistent experience. What do you think of that?

Author: oh why is this here? i don't remember even adding this or maybe i was drafting it before i chatted with you. @matthewshaver should i just remove this since you're tackling this? i don't want to mix different changes into one pr

Author: removing!

> The dbt CLI is a <Constant name="dbt_platform" /> tool available to users on any [available plan](https://www.getdbt.com/pricing). If your dbt Cloud project uses Fusion as its engine, the platform CLI will automatically use Fusion for remote execution — no separate Fusion install is required.

Reviewer: ```suggestion
The dbt CLI is a <Constant name="dbt_platform" /> tool available to users on any [available plan](https://www.getdbt.com/pricing). 

If your <Constant name="dbt_platform" /> project uses <Constant name="Fusion" /> as its engine, the platform CLI will automatically use it for remote execution, so no local <Constant name="Fusion" /> installation is required.
```

> However, if you also have <Constant name="core" /> or the <Constant name="fusion_engine" /> CLI installed locally, note that all three tools use the `dbt` command. You'll need to configure aliases or use explicit binary paths to invoke the right tool. See the [FAQs](#faqs) below for guidance.

Reviewer: ```suggestion
If you have <Constant name="core" /> and/or the <Constant name="fusion_engine" /> CLI installed locally, all three tools use the `dbt` command. You'll need to configure aliases or use explicit binary paths to invoke the right tool. See the [FAQs](#faqs) below for guidance.
```
